tests/test_binary_pointwise_ops.py

```diff
@@ -1028,30 +1028,31 @@ def test_accuracy_where_scalar_other(shape, scalar, dtype):
 def test_accuracy_isclose(shape, dtype, zero_tol, equal_nan, gen_nan):
     # [gen_nan] 1: nan, 2: inf, 3: -inf, 4: inf vs -inf
     rtol = (
-        torch.rand(1, dtype=torch.float32, device="cuda").item() * 0.0001
+        torch.rand(1, dtype=torch.float32, device="musa").item() * 0.0001
         if not zero_tol
         else 0
     )
     if dtype in ALL_FLOAT_DTYPES:
-        inp1 = torch.randn(shape, dtype=dtype, device="cuda")
-        inp2 = torch.randn(shape, dtype=dtype, device="cuda")
+        inp1 = torch.randn(shape, dtype=dtype, device="musa")
+        inp2 = torch.randn(shape, dtype=dtype, device="musa")
         if gen_nan:
             nan_num = torch.full(
                 (1,),
                 float("nan" if gen_nan == 1 else "inf"),
                 dtype=dtype,
-                device="cuda",
+                device="musa",
             )
-            inp1.view(-1)[0] = -nan_num if gen_nan == 3 else nan_num
-            inp2.view(-1)[0] = -nan_num if gen_nan >= 3 else nan_num
+            # FIXME: Neg doesn't support double on torch_musa, so workaround temporarily.
```

Comment:
how it comes that dtype is double?

Reply:
"nan_num" will be in fp64 type when the dtype is fp64 which is iterated from "ALL_FLOAT_DTYPES", in which case torch_musa may complain that "double not supported"

Reply:
I see. I suggest changing ALL_FLOAT_DTYPES to FLOAT_DTYPES to skip tests on torch.float64.